input/templates/checkout-address.json

```diff
@@ -8,20 +8,20 @@
         "addressForm" : {
             "billingAddressDifferentToBillingAddress": true,
             "shippingAddress" : {
-                "salutations" : { "$ref" : "checkout/salutations.json" },
+                "salutations" : { "$ref" : "form/choose-title-1.json" },
```

Comment:
I have moved to `form` both the template to choose the title and the corresponding JSON. As it is appearing in Checkout, My Address Book, Sign Up and many other different concerns, it makes sense to have it separated like in the case of the country selector. I have renamed it as well to make it similar to the country equivalent.